src/main/resources/application.yml

```diff
@@ -49,7 +49,9 @@ nakadi:
       rotationMs: 86400000 # 1 day
   stream:
     timeoutMs: 31536000000 # 1 year :-P
-    default.commitTimeout: 60 # 1 minute
+    default:
```

Comment:
The default is now only visible in the controller, where I used `defaultValue` for the query parameter.